Action1.tkape

```diff
@@ -0,0 +1,17 @@
+Description: Action1 Application Logs
```

Comment:
This needs to go into the Targets folder and into an appropriate subfolder

Reply:
I'm new to Git. I will resolve these and send a new request with the corrections. Thanks!!

Reply:
Sounds good. Let me know if you have any questions. Best of luck!